Merge Conflict/Assets/Scripts/ComponentHandler.cs

```diff
@@ -196,7 +196,10 @@ private void MergeComponents(GameObject staticComponentObject, GameObject dragge
         mergedElement.InstantiateGameObjectAndAddTexture(staticObjectCanvasPosition);
 
         AnimationManager.Instance.PlayMergeAnimation(staticObjectCanvasPosition, mergedElement, element);
-        ExperienceHandler.AddExperiencePoints(mergedElement.salesXP * (Upgrades.MergeXPUpgrade.GetCurrentPercentageOfSalesXP() / 100));
+
+        int mergeXP = Mathf.CeilToInt(mergedElement.GetSalesXP() * (Upgrades.MergeXPUpgrade.GetCurrentPercentageOfSalesXP() / 100f));
+        ExperienceHandler.AddExperiencePoints(mergeXP);
+
         AudioManager.Instance.PlayMergeSound();
 
```

Comment:
Somehow you don't get any money either ! ... is there something wrong with the logic, or is the data not updated?